examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -191,18 +190,18 @@ genesisinfo {"info": "02c37240d51ea3a03278f3ec3935daa1f59724245eb8af3af4528ed77a
+    "powData": "c149e35e030f0f20c149e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20be49e35e030f0f20be49e35e040f0f20be49e35e040f0f20be49e35e040f0f20be49e35e040f0f20be49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bc49e35e040f0f20bc49e35e040f0f20", 
     "mcBlockHeight": 220, 
-    "mcNetwork": "regtest"
+    "mcNetwork": "regtest", 
+    "withdrawalEpochLength": 10 
 }
 ```
 
 **Step 11: Update Sidechain configuration file**
 
-Use `examples/simpleapp/src/main/resources/settings_basic.conf` file and append it with the result from `genesisinfo` in the following structure:
+Use `examples/simpleapp/src/main/resources/settings_basic.conf` file and supplement it with the result from `genesisinfo` in the following structure:
```

Comment:
The file **"examples/simpleapp/src/main/resources/settings_basic.conf"** is mentioned and that it should be supplemented with data. But later the file **"./examples/simpleapp/src/main/resources/my_settings.conf"** is mentioned instead. Is it supposed to be different files?